server/src/containers/docker.js

```diff
@@ -97,12 +107,16 @@ export function buildQuestImage(questPath, questId) {
         }
         response.on('data', (data) => {
           // Process the data (this could be Docker build output)
-          const { stream, error } = JSON.parse(data.toString())
-          if (stream) {
-            console.log(stream)
-          }
-          if (error) {
-            console.error(error)
+          try {
+            data
+              .toString()
+              .split('\n')
+              .map((line) => line.trim())
+              .filter((line) => line !== '')
+              .forEach((line) => {
+                parseJSONOutput(line.trim())
```

Comment:
Sorry，這邊前面 trim 過了不用再 trim 一次
```suggestion
                parseJSONOutput(line)
```